src/api/checkout/checkout_payment_activations/v1/_base_policy.xml.tpl

```diff
@@ -27,7 +27,7 @@
         </set-header>
         <set-variable name="body" value="@(context.Response.Body.As<JObject>())" />
         <choose>
-            <when condition="@( (context.Response.StatusCode == 500 || context.Response.StatusCode == 424) && ((JObject) context.Variables["body"])["detail_v2"] != null )">
+            <when condition="@( (context.Response.StatusCode == 500 || context.Response.StatusCode == 502 || context.Response.StatusCode == 503 || context.Response.StatusCode == 504 || context.Response.StatusCode == 598) && ((JObject) context.Variables["body"])["detail_v2"] != null )">
```

Comment:
in order to have a backwards compatible policy, better leave 424, in this way we can apply this policy without deploy the pagopa-proxy